pkg/pillar/scripts/device-steps.sh

```diff
@@ -468,6 +468,13 @@ if ! pgrep logmanager >/dev/null; then
     touch "$WATCHDOG_FILE/logmanager.touch"
 fi
 
+if ! pgrep loguploader >/dev/null; then
+    echo "$(date -Ins -u) Starting loguploader"
+    $BINDIR/loguploader &
+    wait_for_touch loguploader
+    touch "$WATCHDOG_FILE/loguploader.touch"
```

Comment:
Are you touching loguploader.pid in some existing loop?

Reply:
have a "<-stillRunning.C" in the loguploader's main loop